ua_parser/user_agent_parser.py

```diff
@@ -117,7 +117,10 @@ def Parse(self, user_agent_string):
                 os = match.group(1)
 
             if self.os_v1_replacement:
-                os_v1 = self.os_v1_replacement
+                if re.search(r'\$1', self.os_v1_replacement):
```

Comment:
Mind explaining what this is for? It seems we don't need a regular expression search here, and wouldn't just `'\$1' in self.os_v1_replacement` be sufficient?

Reply:
It is basically a copy of https://github.com/ondras/uap-python/blob/990919b5f49ecb04c126b93123efabad2a95d757/ua_parser/user_agent_parser.py#L112 -- the very same logic that was already present for `os_replacement`. I aimed at code consistency here.

Reply:
Ah, that's fair. I might attempt a follow up to change them to not use regular expressions since that's a bit heavy.